src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/LoopDetector.kt

```diff
@@ -695,13 +666,13 @@ private class ReplayModeLoopDetectorHelper(
     fun initialize() {
         currentInterleavingNodeIndex = 0
         executionsPerformedInCurrentThread = 0
-        threadsRan.clear()
     }
 
     /**
      * Called before next execution in current thread.
      */
-    fun onNextExecution() {
+    @Suppress("UNUSED_PARAMETER")
```

Comment:
Maybe a comment on why we still choose to have this parameter?

Reply:
Done.